subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/actions/TypeText.kt

```diff
@@ -75,7 +75,7 @@ internal class TypeText(private val stringToBeTyped: String) : ViewAction {
             ApplicationProvider.getApplicationContext<Application>()
                 .getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
             )
-            .getFieldByReflectionWithAnyField("mIInputContext")
+            .getFieldByReflectionWithAnyField("mFallbackInputConnection")
```

Comment:
Hey, it should be under SDK check if, right?

Reply:
@Chog0 What do you think could you make changes with SDK check?

Reply:
Sure, will do that and ask you for review when it's ready.
Thanks for pointing this out, guys!

Reply:
I will merge and create a release

Reply:
LGTM

Reply:
Released in 2023.2